DependencyInjection/DoctrineExtension.php

```diff
@@ -793,26 +794,6 @@ protected function loadOrmEntityManager(array $entityManager, ContainerBuilder $
      * @param ContainerBuilder     $container     A ContainerBuilder instance
      *
      * @example
-     *
-     *  doctrine.orm:
```

Comment:
`vendor/bin/phpcbf` removed those :thinking:

Reply:
I suggest reporting that as a bug (after identifying which rule removed it if possible)

Reply:
@stof Is this more an issue on https://github.com/slevomat/coding-standard ? Not that familiar with PHP_CodeSniffer

```
	=> Changeset started by SlevomatCodingStandard.Commenting.DocCommentSpacing:756
		Q: .Helpers.Fix:23 replaced token 6678 (T_DOC_COMMENT_WHITESPACE on line 796) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6679 (T_DOC_COMMENT_WHITESPACE on line 797) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6680 (T_DOC_COMMENT_STAR on line 797) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6681 (T_DOC_COMMENT_WHITESPACE on line 797) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6682 (T_DOC_COMMENT_WHITESPACE on line 798) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6683 (T_DOC_COMMENT_STAR on line 798) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6684 (T_DOC_COMMENT_WHITESPACE on line 798) "··doctrine.orm:" => "doctrine.orm:"
		Q: .Helpers.Fix:23 replaced token 6685 (T_DOC_COMMENT_STRING on line 798) "doctrine.orm:" => ""
		Q: .Helpers.Fix:23 replaced token 6686 (T_DOC_COMMENT_WHITESPACE on line 798) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6687 (T_DOC_COMMENT_WHITESPACE on line 799) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6688 (T_DOC_COMMENT_STAR on line 799) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6689 (T_DOC_COMMENT_WHITESPACE on line 799) "·····mappings:" => "mappings:"
		Q: .Helpers.Fix:23 replaced token 6690 (T_DOC_COMMENT_STRING on line 799) "mappings:" => ""
		Q: .Helpers.Fix:23 replaced token 6691 (T_DOC_COMMENT_WHITESPACE on line 799) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6692 (T_DOC_COMMENT_WHITESPACE on line 800) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6693 (T_DOC_COMMENT_STAR on line 800) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6694 (T_DOC_COMMENT_WHITESPACE on line 800) "·········MyBundle1:·~" => "MyBundle1:·~"
		Q: .Helpers.Fix:23 replaced token 6695 (T_DOC_COMMENT_STRING on line 800) "MyBundle1:·~" => ""
		Q: .Helpers.Fix:23 replaced token 6696 (T_DOC_COMMENT_WHITESPACE on line 800) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6697 (T_DOC_COMMENT_WHITESPACE on line 801) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6698 (T_DOC_COMMENT_STAR on line 801) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6699 (T_DOC_COMMENT_WHITESPACE on line 801) "·········MyBundle2:·yml" => "MyBundle2:·yml"
		Q: .Helpers.Fix:23 replaced token 6700 (T_DOC_COMMENT_STRING on line 801) "MyBundle2:·yml" => ""
		Q: .Helpers.Fix:23 replaced token 6701 (T_DOC_COMMENT_WHITESPACE on line 801) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6702 (T_DOC_COMMENT_WHITESPACE on line 802) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6703 (T_DOC_COMMENT_STAR on line 802) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6704 (T_DOC_COMMENT_WHITESPACE on line 802) "·········MyBundle3:·{·type:·annotation,·dir:·Entities/·}" => "MyBundle3:·{·type:·annotation,·dir:·Entities/·}"
		Q: .Helpers.Fix:23 replaced token 6705 (T_DOC_COMMENT_STRING on line 802) "MyBundle3:·{·type:·annotation,·dir:·Entities/·}" => ""
		Q: .Helpers.Fix:23 replaced token 6706 (T_DOC_COMMENT_WHITESPACE on line 802) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6707 (T_DOC_COMMENT_WHITESPACE on line 803) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6708 (T_DOC_COMMENT_STAR on line 803) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6709 (T_DOC_COMMENT_WHITESPACE on line 803) "·········MyBundle4:·{·type:·xml,·dir:·Resources/config/doctrine/mapping·}" => "MyBundle4:·{·type:·xml,·dir:·Resources/config/doctrine/mapping·}"
		Q: .Helpers.Fix:23 replaced token 6710 (T_DOC_COMMENT_STRING on line 803) "MyBundle4:·{·type:·xml,·dir:·Resources/config/doctrine/mapping·}" => ""
		Q: .Helpers.Fix:23 replaced token 6711 (T_DOC_COMMENT_WHITESPACE on line 803) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6712 (T_DOC_COMMENT_WHITESPACE on line 804) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6713 (T_DOC_COMMENT_STAR on line 804) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6714 (T_DOC_COMMENT_WHITESPACE on line 804) "·········MyBundle5:·{·type:·attribute,·dir:·Entities/·}" => "MyBundle5:·{·type:·attribute,·dir:·Entities/·}"
		Q: .Helpers.Fix:23 replaced token 6715 (T_DOC_COMMENT_STRING on line 804) "MyBundle5:·{·type:·attribute,·dir:·Entities/·}" => ""
		Q: .Helpers.Fix:23 replaced token 6716 (T_DOC_COMMENT_WHITESPACE on line 804) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6717 (T_DOC_COMMENT_WHITESPACE on line 805) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6718 (T_DOC_COMMENT_STAR on line 805) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6719 (T_DOC_COMMENT_WHITESPACE on line 805) "·········MyBundle6:" => "MyBundle6:"
		Q: .Helpers.Fix:23 replaced token 6720 (T_DOC_COMMENT_STRING on line 805) "MyBundle6:" => ""
		Q: .Helpers.Fix:23 replaced token 6721 (T_DOC_COMMENT_WHITESPACE on line 805) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6722 (T_DOC_COMMENT_WHITESPACE on line 806) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6723 (T_DOC_COMMENT_STAR on line 806) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6724 (T_DOC_COMMENT_WHITESPACE on line 806) "·············type:·yml" => "type:·yml"
		Q: .Helpers.Fix:23 replaced token 6725 (T_DOC_COMMENT_STRING on line 806) "type:·yml" => ""
		Q: .Helpers.Fix:23 replaced token 6726 (T_DOC_COMMENT_WHITESPACE on line 806) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6727 (T_DOC_COMMENT_WHITESPACE on line 807) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6728 (T_DOC_COMMENT_STAR on line 807) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6729 (T_DOC_COMMENT_WHITESPACE on line 807) "·············dir:·bundle-mappings/" => "dir:·bundle-mappings/"
		Q: .Helpers.Fix:23 replaced token 6730 (T_DOC_COMMENT_STRING on line 807) "dir:·bundle-mappings/" => ""
		Q: .Helpers.Fix:23 replaced token 6731 (T_DOC_COMMENT_WHITESPACE on line 807) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6732 (T_DOC_COMMENT_WHITESPACE on line 808) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6733 (T_DOC_COMMENT_STAR on line 808) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6734 (T_DOC_COMMENT_WHITESPACE on line 808) "·············alias:·BundleAlias" => "alias:·BundleAlias"
		Q: .Helpers.Fix:23 replaced token 6735 (T_DOC_COMMENT_STRING on line 808) "alias:·BundleAlias" => ""
		Q: .Helpers.Fix:23 replaced token 6736 (T_DOC_COMMENT_WHITESPACE on line 808) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6737 (T_DOC_COMMENT_WHITESPACE on line 809) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6738 (T_DOC_COMMENT_STAR on line 809) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6739 (T_DOC_COMMENT_WHITESPACE on line 809) "·········arbitrary_key:" => "arbitrary_key:"
		Q: .Helpers.Fix:23 replaced token 6740 (T_DOC_COMMENT_STRING on line 809) "arbitrary_key:" => ""
		Q: .Helpers.Fix:23 replaced token 6741 (T_DOC_COMMENT_WHITESPACE on line 809) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6742 (T_DOC_COMMENT_WHITESPACE on line 810) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6743 (T_DOC_COMMENT_STAR on line 810) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6744 (T_DOC_COMMENT_WHITESPACE on line 810) "·············type:·xml" => "type:·xml"
		Q: .Helpers.Fix:23 replaced token 6745 (T_DOC_COMMENT_STRING on line 810) "type:·xml" => ""
		Q: .Helpers.Fix:23 replaced token 6746 (T_DOC_COMMENT_WHITESPACE on line 810) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6747 (T_DOC_COMMENT_WHITESPACE on line 811) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6748 (T_DOC_COMMENT_STAR on line 811) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6749 (T_DOC_COMMENT_WHITESPACE on line 811) "·············dir:·%kernel.project_dir%/src/vendor/DoctrineExtensions/lib/DoctrineExtensions/Entities" => "dir:·%kernel.project_dir%/src/vendor/DoctrineExtensions/lib/DoctrineExtensions/Entities"
		Q: .Helpers.Fix:23 replaced token 6750 (T_DOC_COMMENT_STRING on line 811) "dir:·%kernel.project_dir%/src/vendor/DoctrineExtensions/lib/DoctrineExtensions/Entities" => ""
		Q: .Helpers.Fix:23 replaced token 6751 (T_DOC_COMMENT_WHITESPACE on line 811) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6752 (T_DOC_COMMENT_WHITESPACE on line 812) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6753 (T_DOC_COMMENT_STAR on line 812) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6754 (T_DOC_COMMENT_WHITESPACE on line 812) "·············prefix:·DoctrineExtensions\Entities\" => "prefix:·DoctrineExtensions\Entities\"
		Q: .Helpers.Fix:23 replaced token 6755 (T_DOC_COMMENT_STRING on line 812) "prefix:·DoctrineExtensions\Entities\" => ""
		Q: .Helpers.Fix:23 replaced token 6756 (T_DOC_COMMENT_WHITESPACE on line 812) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6757 (T_DOC_COMMENT_WHITESPACE on line 813) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6758 (T_DOC_COMMENT_STAR on line 813) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6759 (T_DOC_COMMENT_WHITESPACE on line 813) "·············alias:·DExt" => "alias:·DExt"
		Q: .Helpers.Fix:23 replaced token 6760 (T_DOC_COMMENT_STRING on line 813) "alias:·DExt" => ""
		Q: .Helpers.Fix:23 replaced token 6761 (T_DOC_COMMENT_WHITESPACE on line 813) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6762 (T_DOC_COMMENT_WHITESPACE on line 814) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6763 (T_DOC_COMMENT_STAR on line 814) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6764 (T_DOC_COMMENT_WHITESPACE on line 814) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6765 (T_DOC_COMMENT_WHITESPACE on line 815) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6766 (T_DOC_COMMENT_STAR on line 815) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6767 (T_DOC_COMMENT_WHITESPACE on line 815) "·In·the·case·of·bundles·everything·is·really·optional·(which·leads·to·autodetection·for·this·bundle)·but" => "In·the·case·of·bundles·everything·is·really·optional·(which·leads·to·autodetection·for·this·bundle)·but"
		Q: .Helpers.Fix:23 replaced token 6768 (T_DOC_COMMENT_STRING on line 815) "In·the·case·of·bundles·everything·is·really·optional·(which·leads·to·autodetection·for·this·bundle)·but" => ""
		Q: .Helpers.Fix:23 replaced token 6769 (T_DOC_COMMENT_WHITESPACE on line 815) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6770 (T_DOC_COMMENT_WHITESPACE on line 816) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 6771 (T_DOC_COMMENT_STAR on line 816) "*" => ""
		Q: .Helpers.Fix:23 replaced token 6772 (T_DOC_COMMENT_WHITESPACE on line 816) "·in·the·mappings·key·everything·except·alias·is·a·required·argument." => "in·the·mappings·key·everything·except·alias·is·a·required·argument."
		Q: .Helpers.Fix:23 replaced token 6773 (T_DOC_COMMENT_STRING on line 816) "in·the·mappings·key·everything·except·alias·is·a·required·argument." => ""
		Q: .Helpers.Fix:23 replaced token 6774 (T_DOC_COMMENT_WHITESPACE on line 816) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 6775 (T_DOC_COMMENT_WHITESPACE on line 817) "·····*/" => "*/"
		Q: SlevomatCodingStandard.Commenting.DocCommentSpacing:760 replaced token 6677 (T_DOC_COMMENT_TAG on line 796) "@example" => "@example\n"
		Q: SlevomatCodingStandard.Commenting.DocCommentSpacing:766 replaced token 6776 (T_DOC_COMMENT_CLOSE_TAG on line 817) "*/" => "·····*/"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6678 (T_DOC_COMMENT_WHITESPACE on line 796) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6679 (T_DOC_COMMENT_WHITESPACE on line 797) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6680 (T_DOC_COMMENT_STAR on line 797) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6681 (T_DOC_COMMENT_WHITESPACE on line 797) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6682 (T_DOC_COMMENT_WHITESPACE on line 798) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6683 (T_DOC_COMMENT_STAR on line 798) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6684 (T_DOC_COMMENT_WHITESPACE on line 798) "··doctrine.orm:" => "doctrine.orm:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6685 (T_DOC_COMMENT_STRING on line 798) "doctrine.orm:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6686 (T_DOC_COMMENT_WHITESPACE on line 798) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6687 (T_DOC_COMMENT_WHITESPACE on line 799) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6688 (T_DOC_COMMENT_STAR on line 799) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6689 (T_DOC_COMMENT_WHITESPACE on line 799) "·····mappings:" => "mappings:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6690 (T_DOC_COMMENT_STRING on line 799) "mappings:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6691 (T_DOC_COMMENT_WHITESPACE on line 799) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6692 (T_DOC_COMMENT_WHITESPACE on line 800) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6693 (T_DOC_COMMENT_STAR on line 800) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6694 (T_DOC_COMMENT_WHITESPACE on line 800) "·········MyBundle1:·~" => "MyBundle1:·~"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6695 (T_DOC_COMMENT_STRING on line 800) "MyBundle1:·~" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6696 (T_DOC_COMMENT_WHITESPACE on line 800) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6697 (T_DOC_COMMENT_WHITESPACE on line 801) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6698 (T_DOC_COMMENT_STAR on line 801) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6699 (T_DOC_COMMENT_WHITESPACE on line 801) "·········MyBundle2:·yml" => "MyBundle2:·yml"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6700 (T_DOC_COMMENT_STRING on line 801) "MyBundle2:·yml" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6701 (T_DOC_COMMENT_WHITESPACE on line 801) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6702 (T_DOC_COMMENT_WHITESPACE on line 802) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6703 (T_DOC_COMMENT_STAR on line 802) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6704 (T_DOC_COMMENT_WHITESPACE on line 802) "·········MyBundle3:·{·type:·annotation,·dir:·Entities/·}" => "MyBundle3:·{·type:·annotation,·dir:·Entities/·}"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6705 (T_DOC_COMMENT_STRING on line 802) "MyBundle3:·{·type:·annotation,·dir:·Entities/·}" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6706 (T_DOC_COMMENT_WHITESPACE on line 802) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6707 (T_DOC_COMMENT_WHITESPACE on line 803) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6708 (T_DOC_COMMENT_STAR on line 803) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6709 (T_DOC_COMMENT_WHITESPACE on line 803) "·········MyBundle4:·{·type:·xml,·dir:·Resources/config/doctrine/mapping·}" => "MyBundle4:·{·type:·xml,·dir:·Resources/config/doctrine/mapping·}"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6710 (T_DOC_COMMENT_STRING on line 803) "MyBundle4:·{·type:·xml,·dir:·Resources/config/doctrine/mapping·}" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6711 (T_DOC_COMMENT_WHITESPACE on line 803) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6712 (T_DOC_COMMENT_WHITESPACE on line 804) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6713 (T_DOC_COMMENT_STAR on line 804) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6714 (T_DOC_COMMENT_WHITESPACE on line 804) "·········MyBundle5:·{·type:·attribute,·dir:·Entities/·}" => "MyBundle5:·{·type:·attribute,·dir:·Entities/·}"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6715 (T_DOC_COMMENT_STRING on line 804) "MyBundle5:·{·type:·attribute,·dir:·Entities/·}" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6716 (T_DOC_COMMENT_WHITESPACE on line 804) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6717 (T_DOC_COMMENT_WHITESPACE on line 805) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6718 (T_DOC_COMMENT_STAR on line 805) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6719 (T_DOC_COMMENT_WHITESPACE on line 805) "·········MyBundle6:" => "MyBundle6:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6720 (T_DOC_COMMENT_STRING on line 805) "MyBundle6:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6721 (T_DOC_COMMENT_WHITESPACE on line 805) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6722 (T_DOC_COMMENT_WHITESPACE on line 806) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6723 (T_DOC_COMMENT_STAR on line 806) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6724 (T_DOC_COMMENT_WHITESPACE on line 806) "·············type:·yml" => "type:·yml"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6725 (T_DOC_COMMENT_STRING on line 806) "type:·yml" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6726 (T_DOC_COMMENT_WHITESPACE on line 806) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6727 (T_DOC_COMMENT_WHITESPACE on line 807) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6728 (T_DOC_COMMENT_STAR on line 807) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6729 (T_DOC_COMMENT_WHITESPACE on line 807) "·············dir:·bundle-mappings/" => "dir:·bundle-mappings/"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6730 (T_DOC_COMMENT_STRING on line 807) "dir:·bundle-mappings/" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6731 (T_DOC_COMMENT_WHITESPACE on line 807) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6732 (T_DOC_COMMENT_WHITESPACE on line 808) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6733 (T_DOC_COMMENT_STAR on line 808) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6734 (T_DOC_COMMENT_WHITESPACE on line 808) "·············alias:·BundleAlias" => "alias:·BundleAlias"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6735 (T_DOC_COMMENT_STRING on line 808) "alias:·BundleAlias" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6736 (T_DOC_COMMENT_WHITESPACE on line 808) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6737 (T_DOC_COMMENT_WHITESPACE on line 809) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6738 (T_DOC_COMMENT_STAR on line 809) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6739 (T_DOC_COMMENT_WHITESPACE on line 809) "·········arbitrary_key:" => "arbitrary_key:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6740 (T_DOC_COMMENT_STRING on line 809) "arbitrary_key:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6741 (T_DOC_COMMENT_WHITESPACE on line 809) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6742 (T_DOC_COMMENT_WHITESPACE on line 810) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6743 (T_DOC_COMMENT_STAR on line 810) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6744 (T_DOC_COMMENT_WHITESPACE on line 810) "·············type:·xml" => "type:·xml"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6745 (T_DOC_COMMENT_STRING on line 810) "type:·xml" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6746 (T_DOC_COMMENT_WHITESPACE on line 810) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6747 (T_DOC_COMMENT_WHITESPACE on line 811) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6748 (T_DOC_COMMENT_STAR on line 811) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6749 (T_DOC_COMMENT_WHITESPACE on line 811) "·············dir:·%kernel.project_dir%/src/vendor/DoctrineExtensions/lib/DoctrineExtensions/Entities" => "dir:·%kernel.project_dir%/src/vendor/DoctrineExtensions/lib/DoctrineExtensions/Entities"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6750 (T_DOC_COMMENT_STRING on line 811) "dir:·%kernel.project_dir%/src/vendor/DoctrineExtensions/lib/DoctrineExtensions/Entities" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6751 (T_DOC_COMMENT_WHITESPACE on line 811) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6752 (T_DOC_COMMENT_WHITESPACE on line 812) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6753 (T_DOC_COMMENT_STAR on line 812) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6754 (T_DOC_COMMENT_WHITESPACE on line 812) "·············prefix:·DoctrineExtensions\Entities\" => "prefix:·DoctrineExtensions\Entities\"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6755 (T_DOC_COMMENT_STRING on line 812) "prefix:·DoctrineExtensions\Entities\" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6756 (T_DOC_COMMENT_WHITESPACE on line 812) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6757 (T_DOC_COMMENT_WHITESPACE on line 813) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6758 (T_DOC_COMMENT_STAR on line 813) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6759 (T_DOC_COMMENT_WHITESPACE on line 813) "·············alias:·DExt" => "alias:·DExt"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6760 (T_DOC_COMMENT_STRING on line 813) "alias:·DExt" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6761 (T_DOC_COMMENT_WHITESPACE on line 813) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6762 (T_DOC_COMMENT_WHITESPACE on line 814) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6763 (T_DOC_COMMENT_STAR on line 814) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6764 (T_DOC_COMMENT_WHITESPACE on line 814) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6765 (T_DOC_COMMENT_WHITESPACE on line 815) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6766 (T_DOC_COMMENT_STAR on line 815) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6767 (T_DOC_COMMENT_WHITESPACE on line 815) "·In·the·case·of·bundles·everything·is·really·optional·(which·leads·to·autodetection·for·this·bundle)·but" => "In·the·case·of·bundles·everything·is·really·optional·(which·leads·to·autodetection·for·this·bundle)·but"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6768 (T_DOC_COMMENT_STRING on line 815) "In·the·case·of·bundles·everything·is·really·optional·(which·leads·to·autodetection·for·this·bundle)·but" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6769 (T_DOC_COMMENT_WHITESPACE on line 815) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6770 (T_DOC_COMMENT_WHITESPACE on line 816) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6771 (T_DOC_COMMENT_STAR on line 816) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6772 (T_DOC_COMMENT_WHITESPACE on line 816) "·in·the·mappings·key·everything·except·alias·is·a·required·argument." => "in·the·mappings·key·everything·except·alias·is·a·required·argument."
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6773 (T_DOC_COMMENT_STRING on line 816) "in·the·mappings·key·everything·except·alias·is·a·required·argument." => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6774 (T_DOC_COMMENT_WHITESPACE on line 816) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6775 (T_DOC_COMMENT_WHITESPACE on line 817) "·····*/" => "*/"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6677 (T_DOC_COMMENT_TAG on line 796) "@example" => "@example\n"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 6776 (T_DOC_COMMENT_CLOSE_TAG on line 817) "*/" => "·····*/"
	=> Changeset ended: 100 changes applied
	=> Changeset started by SlevomatCodingStandard.Commenting.DocCommentSpacing:756
		Q: .Helpers.Fix:23 replaced token 7092 (T_DOC_COMMENT_WHITESPACE on line 853) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7093 (T_DOC_COMMENT_WHITESPACE on line 854) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7094 (T_DOC_COMMENT_STAR on line 854) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7095 (T_DOC_COMMENT_WHITESPACE on line 854) "··entity_managers:" => "entity_managers:"
		Q: .Helpers.Fix:23 replaced token 7096 (T_DOC_COMMENT_STRING on line 854) "entity_managers:" => ""
		Q: .Helpers.Fix:23 replaced token 7097 (T_DOC_COMMENT_WHITESPACE on line 854) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7098 (T_DOC_COMMENT_WHITESPACE on line 855) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7099 (T_DOC_COMMENT_STAR on line 855) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7100 (T_DOC_COMMENT_WHITESPACE on line 855) "······default:" => "default:"
		Q: .Helpers.Fix:23 replaced token 7101 (T_DOC_COMMENT_STRING on line 855) "default:" => ""
		Q: .Helpers.Fix:23 replaced token 7102 (T_DOC_COMMENT_WHITESPACE on line 855) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7103 (T_DOC_COMMENT_WHITESPACE on line 856) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7104 (T_DOC_COMMENT_STAR on line 856) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7105 (T_DOC_COMMENT_WHITESPACE on line 856) "··········second_level_cache:" => "second_level_cache:"
		Q: .Helpers.Fix:23 replaced token 7106 (T_DOC_COMMENT_STRING on line 856) "second_level_cache:" => ""
		Q: .Helpers.Fix:23 replaced token 7107 (T_DOC_COMMENT_WHITESPACE on line 856) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7108 (T_DOC_COMMENT_WHITESPACE on line 857) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7109 (T_DOC_COMMENT_STAR on line 857) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7110 (T_DOC_COMMENT_WHITESPACE on line 857) "··············region_lifetime:·3600" => "region_lifetime:·3600"
		Q: .Helpers.Fix:23 replaced token 7111 (T_DOC_COMMENT_STRING on line 857) "region_lifetime:·3600" => ""
		Q: .Helpers.Fix:23 replaced token 7112 (T_DOC_COMMENT_WHITESPACE on line 857) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7113 (T_DOC_COMMENT_WHITESPACE on line 858) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7114 (T_DOC_COMMENT_STAR on line 858) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7115 (T_DOC_COMMENT_WHITESPACE on line 858) "··············region_lock_lifetime:·60" => "region_lock_lifetime:·60"
		Q: .Helpers.Fix:23 replaced token 7116 (T_DOC_COMMENT_STRING on line 858) "region_lock_lifetime:·60" => ""
		Q: .Helpers.Fix:23 replaced token 7117 (T_DOC_COMMENT_WHITESPACE on line 858) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7118 (T_DOC_COMMENT_WHITESPACE on line 859) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7119 (T_DOC_COMMENT_STAR on line 859) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7120 (T_DOC_COMMENT_WHITESPACE on line 859) "··············region_cache_driver:·apc" => "region_cache_driver:·apc"
		Q: .Helpers.Fix:23 replaced token 7121 (T_DOC_COMMENT_STRING on line 859) "region_cache_driver:·apc" => ""
		Q: .Helpers.Fix:23 replaced token 7122 (T_DOC_COMMENT_WHITESPACE on line 859) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7123 (T_DOC_COMMENT_WHITESPACE on line 860) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7124 (T_DOC_COMMENT_STAR on line 860) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7125 (T_DOC_COMMENT_WHITESPACE on line 860) "··············log_enabled:·true" => "log_enabled:·true"
		Q: .Helpers.Fix:23 replaced token 7126 (T_DOC_COMMENT_STRING on line 860) "log_enabled:·true" => ""
		Q: .Helpers.Fix:23 replaced token 7127 (T_DOC_COMMENT_WHITESPACE on line 860) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7128 (T_DOC_COMMENT_WHITESPACE on line 861) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7129 (T_DOC_COMMENT_STAR on line 861) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7130 (T_DOC_COMMENT_WHITESPACE on line 861) "··············regions:" => "regions:"
		Q: .Helpers.Fix:23 replaced token 7131 (T_DOC_COMMENT_STRING on line 861) "regions:" => ""
		Q: .Helpers.Fix:23 replaced token 7132 (T_DOC_COMMENT_WHITESPACE on line 861) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7133 (T_DOC_COMMENT_WHITESPACE on line 862) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7134 (T_DOC_COMMENT_STAR on line 862) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7135 (T_DOC_COMMENT_WHITESPACE on line 862) "··················my_service_region:" => "my_service_region:"
		Q: .Helpers.Fix:23 replaced token 7136 (T_DOC_COMMENT_STRING on line 862) "my_service_region:" => ""
		Q: .Helpers.Fix:23 replaced token 7137 (T_DOC_COMMENT_WHITESPACE on line 862) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7138 (T_DOC_COMMENT_WHITESPACE on line 863) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7139 (T_DOC_COMMENT_STAR on line 863) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7140 (T_DOC_COMMENT_WHITESPACE on line 863) "······················type:·service" => "type:·service"
		Q: .Helpers.Fix:23 replaced token 7141 (T_DOC_COMMENT_STRING on line 863) "type:·service" => ""
		Q: .Helpers.Fix:23 replaced token 7142 (T_DOC_COMMENT_WHITESPACE on line 863) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7143 (T_DOC_COMMENT_WHITESPACE on line 864) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7144 (T_DOC_COMMENT_STAR on line 864) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7145 (T_DOC_COMMENT_WHITESPACE on line 864) "······················service·:·"my_service_region"" => "service·:·"my_service_region""
		Q: .Helpers.Fix:23 replaced token 7146 (T_DOC_COMMENT_STRING on line 864) "service·:·"my_service_region"" => ""
		Q: .Helpers.Fix:23 replaced token 7147 (T_DOC_COMMENT_WHITESPACE on line 864) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7148 (T_DOC_COMMENT_WHITESPACE on line 865) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7149 (T_DOC_COMMENT_STAR on line 865) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7150 (T_DOC_COMMENT_WHITESPACE on line 865) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7151 (T_DOC_COMMENT_WHITESPACE on line 866) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7152 (T_DOC_COMMENT_STAR on line 866) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7153 (T_DOC_COMMENT_WHITESPACE on line 866) "··················my_query_region:" => "my_query_region:"
		Q: .Helpers.Fix:23 replaced token 7154 (T_DOC_COMMENT_STRING on line 866) "my_query_region:" => ""
		Q: .Helpers.Fix:23 replaced token 7155 (T_DOC_COMMENT_WHITESPACE on line 866) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7156 (T_DOC_COMMENT_WHITESPACE on line 867) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7157 (T_DOC_COMMENT_STAR on line 867) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7158 (T_DOC_COMMENT_WHITESPACE on line 867) "······················lifetime:·300" => "lifetime:·300"
		Q: .Helpers.Fix:23 replaced token 7159 (T_DOC_COMMENT_STRING on line 867) "lifetime:·300" => ""
		Q: .Helpers.Fix:23 replaced token 7160 (T_DOC_COMMENT_WHITESPACE on line 867) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7161 (T_DOC_COMMENT_WHITESPACE on line 868) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7162 (T_DOC_COMMENT_STAR on line 868) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7163 (T_DOC_COMMENT_WHITESPACE on line 868) "······················cache_driver:·array" => "cache_driver:·array"
		Q: .Helpers.Fix:23 replaced token 7164 (T_DOC_COMMENT_STRING on line 868) "cache_driver:·array" => ""
		Q: .Helpers.Fix:23 replaced token 7165 (T_DOC_COMMENT_WHITESPACE on line 868) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7166 (T_DOC_COMMENT_WHITESPACE on line 869) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7167 (T_DOC_COMMENT_STAR on line 869) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7168 (T_DOC_COMMENT_WHITESPACE on line 869) "······················type:·filelock" => "type:·filelock"
		Q: .Helpers.Fix:23 replaced token 7169 (T_DOC_COMMENT_STRING on line 869) "type:·filelock" => ""
		Q: .Helpers.Fix:23 replaced token 7170 (T_DOC_COMMENT_WHITESPACE on line 869) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7171 (T_DOC_COMMENT_WHITESPACE on line 870) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7172 (T_DOC_COMMENT_STAR on line 870) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7173 (T_DOC_COMMENT_WHITESPACE on line 870) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7174 (T_DOC_COMMENT_WHITESPACE on line 871) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7175 (T_DOC_COMMENT_STAR on line 871) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7176 (T_DOC_COMMENT_WHITESPACE on line 871) "··················my_entity_region:" => "my_entity_region:"
		Q: .Helpers.Fix:23 replaced token 7177 (T_DOC_COMMENT_STRING on line 871) "my_entity_region:" => ""
		Q: .Helpers.Fix:23 replaced token 7178 (T_DOC_COMMENT_WHITESPACE on line 871) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7179 (T_DOC_COMMENT_WHITESPACE on line 872) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7180 (T_DOC_COMMENT_STAR on line 872) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7181 (T_DOC_COMMENT_WHITESPACE on line 872) "······················lifetime:·600" => "lifetime:·600"
		Q: .Helpers.Fix:23 replaced token 7182 (T_DOC_COMMENT_STRING on line 872) "lifetime:·600" => ""
		Q: .Helpers.Fix:23 replaced token 7183 (T_DOC_COMMENT_WHITESPACE on line 872) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7184 (T_DOC_COMMENT_WHITESPACE on line 873) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7185 (T_DOC_COMMENT_STAR on line 873) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7186 (T_DOC_COMMENT_WHITESPACE on line 873) "······················cache_driver:" => "cache_driver:"
		Q: .Helpers.Fix:23 replaced token 7187 (T_DOC_COMMENT_STRING on line 873) "cache_driver:" => ""
		Q: .Helpers.Fix:23 replaced token 7188 (T_DOC_COMMENT_WHITESPACE on line 873) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7189 (T_DOC_COMMENT_WHITESPACE on line 874) "·····*" => "*"
		Q: .Helpers.Fix:23 replaced token 7190 (T_DOC_COMMENT_STAR on line 874) "*" => ""
		Q: .Helpers.Fix:23 replaced token 7191 (T_DOC_COMMENT_WHITESPACE on line 874) "··························type:·apc" => "type:·apc"
		Q: .Helpers.Fix:23 replaced token 7192 (T_DOC_COMMENT_STRING on line 874) "type:·apc" => ""
		Q: .Helpers.Fix:23 replaced token 7193 (T_DOC_COMMENT_WHITESPACE on line 874) "\n·····" => "·····"
		Q: .Helpers.Fix:23 replaced token 7194 (T_DOC_COMMENT_WHITESPACE on line 875) "·····*/" => "*/"
		Q: SlevomatCodingStandard.Commenting.DocCommentSpacing:760 replaced token 7091 (T_DOC_COMMENT_TAG on line 853) "@example" => "@example\n"
		Q: SlevomatCodingStandard.Commenting.DocCommentSpacing:766 replaced token 7195 (T_DOC_COMMENT_CLOSE_TAG on line 875) "*/" => "·····*/"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7092 (T_DOC_COMMENT_WHITESPACE on line 853) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7093 (T_DOC_COMMENT_WHITESPACE on line 854) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7094 (T_DOC_COMMENT_STAR on line 854) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7095 (T_DOC_COMMENT_WHITESPACE on line 854) "··entity_managers:" => "entity_managers:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7096 (T_DOC_COMMENT_STRING on line 854) "entity_managers:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7097 (T_DOC_COMMENT_WHITESPACE on line 854) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7098 (T_DOC_COMMENT_WHITESPACE on line 855) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7099 (T_DOC_COMMENT_STAR on line 855) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7100 (T_DOC_COMMENT_WHITESPACE on line 855) "······default:" => "default:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7101 (T_DOC_COMMENT_STRING on line 855) "default:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7102 (T_DOC_COMMENT_WHITESPACE on line 855) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7103 (T_DOC_COMMENT_WHITESPACE on line 856) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7104 (T_DOC_COMMENT_STAR on line 856) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7105 (T_DOC_COMMENT_WHITESPACE on line 856) "··········second_level_cache:" => "second_level_cache:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7106 (T_DOC_COMMENT_STRING on line 856) "second_level_cache:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7107 (T_DOC_COMMENT_WHITESPACE on line 856) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7108 (T_DOC_COMMENT_WHITESPACE on line 857) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7109 (T_DOC_COMMENT_STAR on line 857) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7110 (T_DOC_COMMENT_WHITESPACE on line 857) "··············region_lifetime:·3600" => "region_lifetime:·3600"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7111 (T_DOC_COMMENT_STRING on line 857) "region_lifetime:·3600" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7112 (T_DOC_COMMENT_WHITESPACE on line 857) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7113 (T_DOC_COMMENT_WHITESPACE on line 858) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7114 (T_DOC_COMMENT_STAR on line 858) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7115 (T_DOC_COMMENT_WHITESPACE on line 858) "··············region_lock_lifetime:·60" => "region_lock_lifetime:·60"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7116 (T_DOC_COMMENT_STRING on line 858) "region_lock_lifetime:·60" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7117 (T_DOC_COMMENT_WHITESPACE on line 858) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7118 (T_DOC_COMMENT_WHITESPACE on line 859) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7119 (T_DOC_COMMENT_STAR on line 859) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7120 (T_DOC_COMMENT_WHITESPACE on line 859) "··············region_cache_driver:·apc" => "region_cache_driver:·apc"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7121 (T_DOC_COMMENT_STRING on line 859) "region_cache_driver:·apc" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7122 (T_DOC_COMMENT_WHITESPACE on line 859) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7123 (T_DOC_COMMENT_WHITESPACE on line 860) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7124 (T_DOC_COMMENT_STAR on line 860) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7125 (T_DOC_COMMENT_WHITESPACE on line 860) "··············log_enabled:·true" => "log_enabled:·true"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7126 (T_DOC_COMMENT_STRING on line 860) "log_enabled:·true" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7127 (T_DOC_COMMENT_WHITESPACE on line 860) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7128 (T_DOC_COMMENT_WHITESPACE on line 861) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7129 (T_DOC_COMMENT_STAR on line 861) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7130 (T_DOC_COMMENT_WHITESPACE on line 861) "··············regions:" => "regions:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7131 (T_DOC_COMMENT_STRING on line 861) "regions:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7132 (T_DOC_COMMENT_WHITESPACE on line 861) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7133 (T_DOC_COMMENT_WHITESPACE on line 862) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7134 (T_DOC_COMMENT_STAR on line 862) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7135 (T_DOC_COMMENT_WHITESPACE on line 862) "··················my_service_region:" => "my_service_region:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7136 (T_DOC_COMMENT_STRING on line 862) "my_service_region:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7137 (T_DOC_COMMENT_WHITESPACE on line 862) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7138 (T_DOC_COMMENT_WHITESPACE on line 863) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7139 (T_DOC_COMMENT_STAR on line 863) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7140 (T_DOC_COMMENT_WHITESPACE on line 863) "······················type:·service" => "type:·service"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7141 (T_DOC_COMMENT_STRING on line 863) "type:·service" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7142 (T_DOC_COMMENT_WHITESPACE on line 863) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7143 (T_DOC_COMMENT_WHITESPACE on line 864) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7144 (T_DOC_COMMENT_STAR on line 864) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7145 (T_DOC_COMMENT_WHITESPACE on line 864) "······················service·:·"my_service_region"" => "service·:·"my_service_region""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7146 (T_DOC_COMMENT_STRING on line 864) "service·:·"my_service_region"" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7147 (T_DOC_COMMENT_WHITESPACE on line 864) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7148 (T_DOC_COMMENT_WHITESPACE on line 865) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7149 (T_DOC_COMMENT_STAR on line 865) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7150 (T_DOC_COMMENT_WHITESPACE on line 865) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7151 (T_DOC_COMMENT_WHITESPACE on line 866) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7152 (T_DOC_COMMENT_STAR on line 866) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7153 (T_DOC_COMMENT_WHITESPACE on line 866) "··················my_query_region:" => "my_query_region:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7154 (T_DOC_COMMENT_STRING on line 866) "my_query_region:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7155 (T_DOC_COMMENT_WHITESPACE on line 866) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7156 (T_DOC_COMMENT_WHITESPACE on line 867) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7157 (T_DOC_COMMENT_STAR on line 867) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7158 (T_DOC_COMMENT_WHITESPACE on line 867) "······················lifetime:·300" => "lifetime:·300"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7159 (T_DOC_COMMENT_STRING on line 867) "lifetime:·300" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7160 (T_DOC_COMMENT_WHITESPACE on line 867) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7161 (T_DOC_COMMENT_WHITESPACE on line 868) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7162 (T_DOC_COMMENT_STAR on line 868) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7163 (T_DOC_COMMENT_WHITESPACE on line 868) "······················cache_driver:·array" => "cache_driver:·array"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7164 (T_DOC_COMMENT_STRING on line 868) "cache_driver:·array" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7165 (T_DOC_COMMENT_WHITESPACE on line 868) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7166 (T_DOC_COMMENT_WHITESPACE on line 869) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7167 (T_DOC_COMMENT_STAR on line 869) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7168 (T_DOC_COMMENT_WHITESPACE on line 869) "······················type:·filelock" => "type:·filelock"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7169 (T_DOC_COMMENT_STRING on line 869) "type:·filelock" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7170 (T_DOC_COMMENT_WHITESPACE on line 869) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7171 (T_DOC_COMMENT_WHITESPACE on line 870) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7172 (T_DOC_COMMENT_STAR on line 870) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7173 (T_DOC_COMMENT_WHITESPACE on line 870) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7174 (T_DOC_COMMENT_WHITESPACE on line 871) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7175 (T_DOC_COMMENT_STAR on line 871) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7176 (T_DOC_COMMENT_WHITESPACE on line 871) "··················my_entity_region:" => "my_entity_region:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7177 (T_DOC_COMMENT_STRING on line 871) "my_entity_region:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7178 (T_DOC_COMMENT_WHITESPACE on line 871) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7179 (T_DOC_COMMENT_WHITESPACE on line 872) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7180 (T_DOC_COMMENT_STAR on line 872) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7181 (T_DOC_COMMENT_WHITESPACE on line 872) "······················lifetime:·600" => "lifetime:·600"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7182 (T_DOC_COMMENT_STRING on line 872) "lifetime:·600" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7183 (T_DOC_COMMENT_WHITESPACE on line 872) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7184 (T_DOC_COMMENT_WHITESPACE on line 873) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7185 (T_DOC_COMMENT_STAR on line 873) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7186 (T_DOC_COMMENT_WHITESPACE on line 873) "······················cache_driver:" => "cache_driver:"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7187 (T_DOC_COMMENT_STRING on line 873) "cache_driver:" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7188 (T_DOC_COMMENT_WHITESPACE on line 873) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7189 (T_DOC_COMMENT_WHITESPACE on line 874) "·····*" => "*"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7190 (T_DOC_COMMENT_STAR on line 874) "*" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7191 (T_DOC_COMMENT_WHITESPACE on line 874) "··························type:·apc" => "type:·apc"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7192 (T_DOC_COMMENT_STRING on line 874) "type:·apc" => ""
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7193 (T_DOC_COMMENT_WHITESPACE on line 874) "\n·····" => "·····"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7194 (T_DOC_COMMENT_WHITESPACE on line 875) "·····*/" => "*/"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7091 (T_DOC_COMMENT_TAG on line 853) "@example" => "@example\n"
		A: SlevomatCodingStandard.Commenting.DocCommentSpacing:768 replaced token 7195 (T_DOC_COMMENT_CLOSE_TAG on line 875) "*/" => "·····*/"
	=> Changeset ended: 105 changes applied

Reply:
Based on those logs, my guess is indeed that the buggy behavior comes from a rule provided by SlevomatCodingStandard and so this should be where the issue should be reported.

Reply:
See https://github.com/slevomat/coding-standard/issues/1579